content/home/examples/a-simple-component.md

```diff
@@ -1,9 +1,12 @@
 ---
-title: A Simple Component
+title: Een Eenvoudige Component
 order: 0
 domid: hello-example
 ---
 
-React components implement a `render()` method that takes input data and returns what to display. This example uses an XML-like syntax called JSX. Input data that is passed into the component can be accessed by `render()` via `this.props`.
+React componenten implementeren een `render()` functie die input data inneemt en terug geeft wat er weergegeven moet worden.
+Dit voorbeeld gebruikt de XML-achtige syntax genaamd JSX. 
+Input data die wordt doorgegeven aan het component is voor `render()` toegankelijk via `this.props`.
```

Comment:
```suggestion
Input data die wordt doorgegeven aan het component is in de `render()` functie toegankelijk via `this.props`.
```

Reply:
I think both versions are fine.

Reply:
Mine is probably more suited towards experienced developers that know what it means, but @ronderksen's version is more clear for beginners as well. I will take the suggestion